test/amqp-integration-test.py

```diff
@@ -222,14 +223,32 @@ def cleanup():
     else:
         print("\n\nFAILURE %d" % exit_status)
 
-
 exit_status = ExitStatus.OK
 tempdir = tempfile.mkdtemp()
+
+parser = argparse.ArgumentParser(description='Run integration tests')
+parser.add_argument('--no-node', dest="run_node", action="store_false",
```

Comment:
I keep thinking about it and I really don't like this. It means having to go from nice booleans to doing a comma split and dict to detect what was in there for no gain. I don't see how it would be more flexible or clear. It'd just be another flag for folks to add.

Can I not do it?